orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -183,6 +186,9 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 			try {
 				logger.info("Attempting to infer " + reactionInst);
 				ReactionInferrer.inferReaction(reactionInst);
+			} catch (RuntimeException e) {
+				e.printStackTrace();
+				return;
```

Comment:
Just curious - why does this RuntimException catch block have a return but the Exception block doesn't?

Reply:
Good point